src/source.rs

```diff
@@ -4,9 +4,9 @@
 
 use std::cell::RefCell;
 use std::mem::transmute;
-#[cfg(any(feature = "v2_36", unix))]
+#[cfg(any(all(feature = "v2_36", unix), feature = "dox"))]
 use std::os::unix::io::RawFd;
-#[cfg(any(feature = "v2_36", windows))]
+#[cfg(all(feature = "v2_36", windows))]
 use libc::c_int as RawFd;
```

Comment:
Seems this will be defined twice if `features `dox v2_36` on windows.

Reply:
Should be fixed now, thanks.